docs/host/host.md

```diff
@@ -6,7 +6,7 @@ title: ""
 
 # Host Management
 
-Users can view and manage Harvester nodes from the host page. The first node always defaults to be a management node of the cluster. When there are three or more nodes, the two other nodes that first joined are automatically promoted to management nodes to form a HA cluster.
+Users can view and manage Harvester nodes from the host page. The first node always defaults to be a management node of the cluster. When there are three or more nodes, the two other nodes that first joined are automatically promoted to management nodes to form a HA cluster. If there are nodes with the zone label `topology.kubernetes.io/zone`, the node promotion mechanism will make sure that all management nodes will be distributed in three different zones.
```

Comment:
do we need an example of the installation steps? otherwise, the node promotion may already be completed after users visit the host page.

Reply:
How about this:
If there are nodes with the zone label `topology.kubernetes.io/zone` (can be added by [os.labels](https://docs.harvesterhci.io/next/install/harvester-configuration#oslabels) in Harvester configuration file during manual or automatic installation), the node promotion mechanism will make sure that all management nodes will be distributed in three different zones.

Reply:
updated

Reply:
installation steps contain zone labels are very flexible

Reply:
`distributed in` should be `distributed to`